### DISCUSSION TITLE
Attribute for player photos/headshots?

### DISCUSSION THREAD

**Discussion by mpolowski** [cwendt94/espn-api#618] [Q&A]
Hey all, I was wondering if there was a way to pull down the player's associated headshot?

**Comment by cwendt94** [comment]
I do not see it being referenced at all in the player data being returned. On the ESPN's website it seems like they might take the players id and then just use it with this url
Saquons playerId 3929630
https://a.espncdn.com/combiner/i?img=/i/headshots/nfl/players/full/3929630.png&w=96&h=70&cb=1
